tests/integration/helm_default_service_monitors_test.go

```diff
@@ -0,0 +1,157 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal"
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/ctxopts"
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/stepfuncs"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/kubernetes/scheme"
+
+	terrak8s "github.com/gruntwork-io/terratest/modules/k8s"
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+func Test_Helm_Default_Service_Monitors(t *testing.T) {
+	const (
+		tickDuration   = 3 * time.Second
+		waitDuration   = 5 * time.Minute
+		scrapeLabel    = "sumologic.com/scrape"
+		componentLabel = "sumologic.com/component"
+		appLabel       = "sumologic.com/app"
+		eventsAppLabel = "otelcol-events"
+	)
+
+	if err := promv1.AddToScheme(scheme.Scheme); err != nil {
+		require.Fail(t, "failed to register scheme: %v", err)
+	}
+
+	expectedDefaultServiceMonitors := internal.DefaultExpectedSumoServiceMonitors
+
+	featServiceMonitors := features.New("sumo service monitors number").
```

Comment:
All of these servicemonitor tests should be template tests instead in my opinion. You don't need a cluster to check this, just the rendered manifests.